apps/backoffice-v2/src/lib/blocks/hooks/useAssociatedCompaniesInformationBlock/useAssociatedCompaniesInformationBlock.tsx

```diff
@@ -30,7 +30,7 @@ export const useAssociatedCompaniesInformationBlock = (workflows: Array<TWorkflo
                 })
                 .addCell({
                   type: 'subheading',
-                  value: 'User-provided data',
+                  value: 'User-Provided Data',
```

Comment:
Are you sure that's how hyphenated words are capitalized?

Reply:
I'm sure that's how Nitzan wanted it 🤷